lib/matplotlib/tests/test_colors.py

```diff
@@ -208,7 +208,9 @@ def test_Normalize():
     # and for scalar ones.
     eps = np.finfo(np.longdouble).resolution
     norm = plt.Normalize(1, 1 + 100 * eps)
-    assert_equal(norm(1 + 50 * eps), .5)
+    # This returns exactly 0.5 when longdouble is extended precision (80-bit),
+    # but only a value close to it when it is quadruple precision (128-bit).
+    assert 0 < norm(1 + 50 * eps) < 1
```

Comment:
Ah, I was thinking about the default of assert_allclose.  Anyways, see the discussion starting at https://github.com/matplotlib/matplotlib/issues/7159#issuecomment-249054207: the point is that I'd rather let a FP expert confirm what is the tolerance that makes sense; without that the safest is just to check that the normalization is not trivial (does not map everything to 0 or 1).